src/Tizen.Sensor/Tizen.Sensor/SensorRecorderDataManager.cs

```diff
@@ -0,0 +1,576 @@
+            }
+            return result;
+        }
+
+        private bool IsSupported()
+        {
+            bool isSupported;
+            int error = Interop.SensorRecoder.IsSupported(sensorType, out isSupported);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error checking if sensor recorder is supported");
+                throw SensorErrorFactory.CheckAndThrowException(error, "SensorRecorderDataManager.IsSupported Failed");
+            }
+            return isSupported;
+        }
+
+        /// <summary>
+        /// Gets a double value from a record data.
+        /// </summary>
+        /// <param name="key">
+        /// Data attribute to retrieve.
+        /// </param>
+        /// <returns>
+        /// returns a double value from a record data.
+        /// </returns>
+        /// <since_tizen> 6 </since_tizen>
+        public double DataGetDouble(RecorderData key)
+        {
+            double value = -1;
+            if (CheckDataHandle())
+            {
+                int error = Interop.SensorRecoder.DataGetDouble(dataHandle, key, out value);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error getting the sensor recorder double data");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "SensorRecorderDataManager.DataGetDouble Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder data is not created");
+            }
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets the start and the end time of the time period of a given record data.
+        /// </summary>
+        /// <returns>
+        /// returns the start and the end time of the time period of a given record data.
+        /// </returns>
+        /// <since_tizen> 6 </since_tizen>
+        public RecordTimePeriod DataGetTime()
+        {
+            long startTime = -1, endTime = -1;
+            if (CheckDataHandle())
+            {
+                int error = Interop.SensorRecoder.DataGetTime(dataHandle, out startTime, out endTime);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error getting the sensor recorder time");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "SensorRecorderDataManager.DataGetTime Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder data is not created");
+            }
+
+            return new RecordTimePeriod(startTime, endTime);
+        }
+
+        /// <summary>
+        ///  Queries the recorded data synchronously and updates the data handle
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Update(Query query)
```

Comment:
Parameters of methods must have valid documentation. Method has 1 parameter, 0 are documented.